source/annex/macro.rst

```diff
@@ -1,183 +1,21 @@
-energy in terms of monetary value :math:`{PRODENE}_{r, s, y}` as specified in the production function of MACRO.  
-
-.. equation {ENERGY_SUPPLY}_{r, s, y}
-
-.. math:: {PHYSENE}_{r, s, y} \geq {PRODENE}_{r, s, y} \cdot {aeei\_factor}_{r, s, y} \qquad \forall{ r, s, y > 1} 
-
-The cumulative effect of autonomous energy efficiency improvements (AEEI) is captured in :math:`{aeei\_factor}_{r,s,y} = {aeei\_factor}_{r, s, y-1} \cdot (1 - {aeei}_{r,s,y})^{duration\_period}_{y}` 
-with :math:`{aeei\_factor}_{r,s,y=1} = 1`. Therefore, choosing the :math:`{aeei}_{r,s,y}` coefficients appropriately offers the possibility to calibrate MACRO to a certain energy demand trajectory 
-from MESSAGE.
-
-Energy system costs
-~~~~
-Energy system costs are based on a previous MESSAGE model run. The approximation of energy system costs in vicinity of the MESSAGE solution are approximated by a Taylor expansion with the 
-first order term using shadow prices :math:`eneprice_{s, y, r}` of the MESSAGE model's solution and a quadratic second-order term.
-
-.. equation {COST_ENERGY}_{r, y}
-
-.. math:: {EC}_{r, y} = {total\_cost}_{y, r} + \displaystyle \sum_{s} {eneprice}_{s, y, r} \cdot \left( {PHYSENE}_{r, s, y} - {enestart}_{s, y, r} \right) \\
-	+ \displaystyle \sum_{s} \frac{{eneprice}_{s, y, r}}{{enestart}_{s, y, r}} \cdot \left( {PHYSENE}_{r, s, y} - {enestart}_{s, y, r} \right)^2 \qquad \forall{ r, y > 1} 
-
-Finite time horizon correction
-~~~~
-Given the finite time horizon of MACRO, a terminal constraint needs to be applied to ensure that investments are chosen at an appropriate level, i.e. to replace depriciated capital and
-provide net growth of capital stock beyond MACRO's time horizon :cite:`manne_buying_1992`. The goal is to avoid to the extend possible model artifacts resulting from this finite time horizon 
-cutoff.
-
-.. equation {TERMINAL_CONSTRAINT}_{r, y}
-
-.. math:: K_{r, y} \cdot  \left( grow_{r, y} + depr_r \right) \leq I_{r, y} \qquad \forall{ r, y = last year} 
+Finally, the parameterization of MACRO has changed in a specific way. As mentioned, the model's most important input parameters are the projected growth rates of total labor, i.e., the combined effect of labor force and labor productivity growth (note that labor supply growth is also referred to as reference or potential GDP growth) and the annual rates of reference energy intensity reductions, i.e. the so-called autonomous energy efficiecy improvement (AEEI) coefficients. In all recent applications of MACRO, including the Shared Socio-economic Pathways (SSPs), these are calibrated to be consistent with the developments in a MESSAGE scenario. In practice, this happens by running MACRO and adjusting the potential GDP growth rates and the AEEI coefficients on a sectoral basis until MACRO does not produce an energy demand response and GDP feedback compared to the MESSAGE scenario that it is calibrated to.
 
 MACRO parameterization
-----
+----------------------
 
 Initial conditions
```

Comment:
Section on initial conditions is not specific to global model (part of GAMS code) and therefore should be moved to MACRO part of MESSAGEix documentation.

Reply:
Agreed here. It may be worthwhile to:

1) move this to the MACRO docs
2) refer to it
3) provide a table of values (as done elsewhere) for each of the 3 SSPs for the global model